src/helpers/projectHelpers.ts

```diff
@@ -33,11 +34,13 @@ import {
   repairBadTokens,
 } from './tokenHelpers/projects/chainlifeHelpers';
 import { getWeb3 } from '../web3/provider';
-import { getProcessMintFunction } from './tokenHelpers';
+import { getProcessMintFunction, getProcessManyMintsFunction } from './tokenHelpers';
 import { updateMathareDescriptions } from './tokenHelpers/projects/mathareHelpers';
+import { getViemClient } from '../web3/viemClient';
 
 const processNewProjects = async (projects: IProject[], conn: Connection) => {
   // try to add all projects to db, duplicates removed
+  // REVIEW -- If the tx counts change, this will not update the project. addProject only saves a document to the db if the project doesn't already exist.
```

Comment:
I think I have a process to update tx count elsewhere but maybe it would be better to do here with a proper Mongo method.

Reply:
I think I found that after the fact.

Reply:
And forgot to update the comment